src/module/item/item.js

```diff
@@ -10,6 +10,27 @@ import { Mix } from "../utils/custom-mixer.js";
 import { ItemActivationMixin } from "./mixins/item-activation.js";
 import { ItemCapacityMixin } from "./mixins/item-capacity.js";
 
+export const uncountableActivations = Object.freeze([
+    "", // no activation
+    "none", // free
+    "action",
+    "move",
+    "swift",
+    "full",
+    "reaction",
+    "special"
+]);
+
+export const uncountableDurations = Object.freeze([
+    "instantaneous",
+    "permanent",
+    "text"
+]);
+
+export const spellAttackActions = Object.freeze([ "msak", "rsak" ]);
+export const weaponAttackActions = Object.freeze([ "mwak", "rwak" ]);
+export const attackActions = Object.freeze([ "mwak", "rwak", "msak", "rsak" ]);
+
```

Comment:
I think these might be better exposed globally in `CONFIG.SFRPG` as part of `config.js`. You could also probably get away with doing something like `const spellAttackActions = weaponAttackActions`. It doesn't matter that they're the same object because they're frozen and used purely to refer to.

Reply:
That works for me. I've moved those out to `config.js`. I guess one thing to clarify is if that config should be accessed through the global `CONFIG.SFRPG`, or by `import { SFRPG } from "config.js"`. I saw both in use, but elected to use the latter because it was already in use for most files I was modifying.

Reply:
Ok, to clarify this, I've implemented a `d.ts` file for CONFIG.SFRPG so it now has correct intellisense. I think that'd be preferred now because you can access it without an import like `SFRPG`.

This is on the v13 branch so you won't see the intellisense immediately, but when we shift over you will